include/zephyr/xen/public/event_channel.h

```diff
@@ -74,6 +74,62 @@
 typedef uint32_t evtchn_port_t;
 DEFINE_XEN_GUEST_HANDLE(evtchn_port_t);
 
+/*
+ * EVTCHNOP_alloc_unbound: Allocate a port in domain <dom> and mark as
+ * accepting interdomain bindings from domain <remote_dom>. A fresh port
+ * is allocated in <dom> and returned as <port>.
+ * NOTES:
+ *  1. If the caller is unprivileged then <dom> must be DOMID_SELF.
+ *  2. <remote_dom> may be DOMID_SELF, allowing loopback connections.
+ */
+struct evtchn_alloc_unbound {
+	/* IN parameters */
+	domid_t dom, remote_dom;
+	/* OUT parameters */
+	evtchn_port_t port;
+};
```

Comment:
this is not using Doxygen docstrings either, `/**`

Reply:
@gmarull The headers in `xen/public/` have been taken from Xen directly (it is the public interface). There are effort to move to Doxygen (see https://lore.kernel.org/xen-devel/20210705105103.14509-1-luca.fancellu@arm.com/) but this is not yet merged.

I would highly recommend to keep them as-is so it is easier for you to update them in the future. You could then re-sync once the headers have been converted to Doxygen.

Reply:
ok, fair enough, thanks for the details